tests/entrypoints/openai/test_chat.py

```diff
@@ -486,6 +486,46 @@ async def test_chat_completion_stream_options(client: openai.AsyncOpenAI,
     assert last_completion_tokens == 10
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "model_name",
+    ["HuggingFaceH4/zephyr-7b-beta", "zephyr-lora"],
+)
+async def test_chat_with_enable_force_include_usage(client: openai.AsyncOpenAI,
+                                                    model_name: str):
+    messages = [{
+        "role": "system",
+        "content": "You are a helpful assistant."
+    }, {
+        "role": "user",
+        "content": "What is the capital of France?"
+    }]
+
+    stream = await client.chat.completions.create(
+        model=model_name,
+        messages=messages,
+        max_completion_tokens=10,
+        extra_body=dict(min_tokens=10),
+        temperature=0.0,
+        stream=True,
+        enable_force_include_usage=True,
```

Comment:
```suggestion
```

Reply:
We added a `extra_server_args` marker to be able to easily parameterize the `server` fixture without modifying all other tests